external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchStream.scala

```diff
@@ -73,6 +73,10 @@ private[kafka010] class KafkaMicroBatchStream(
     Utils.timeStringAsMs(Option(options.get(
       KafkaSourceProvider.MAX_TRIGGER_DELAY)).getOrElse(DEFAULT_MAX_TRIGGER_DELAY))
 
+  // this allows us to mock system clock for testing purposes
+  private[kafka010] val clock: Clock
+  = if (options.containsKey(MOCK_SYSTEM_TIME)) new MockedSystemClock else new SystemClock
```

Comment:
I suspect the indentation is not per style guide - https://github.com/databricks/scala-style-guide#spacing-and-indentation

Reply:
+1 `=` should not be in same indentation with `p`. If it's the only way to fit L78 to a single line, please use multi-lines instead.

```
private[kafka010] val clock: Clock = if (options.containsKey(MOCK_SYSTEM_TIME)) {
  new MockedSystemClock
} else {
  new SystemClock
}
```

Reply:
will fix